### ISSUE TITLE
Feature request: Make groupfolder quota(-limit) visible to users

### ISSUE THREAD

**Issue by ghost** [nextcloud/groupfolders#288]
Hi!

It would be a nice feature to make the remaining quota (or quota-limit) of the group folder visible for all users.

In the nextcloud files area in the bottom left corner there it shows the users homefolder quota. If the nextcloud users only have e.g. 100 MB of private filespace, but 5 GB inside a groupfolder, at the moment they always see the limit of 100 MB.  One way could be by changing this quota-bar to the quota of the groupfolder, when You browse inside the groupfolder.

Currently, just admins are able to see the quota-limit of the groupfolder. The nextcloud desktop client at least shows the sum of the home folder quota and all the accessed group folders quotas, even if this value gives no information about where You have the filespace, it helps You thinking about how much filespace is left.

Thank You in advance!

**Comment by noresistence** [comment]
In our installation, users only get a small private space, and groupfolders are the main component for data sharing and quota management.
Our users furthermore need to be able to see the quota so that they can notify the admins if the quota of that folder is about to be exceeded. They should be able to do this before starting to upload some Gigabytes of data and then failing due to exceeded quota.

**Comment by cClaude** [comment]
Look like a duplicate of https://github.com/nextcloud/groupfolders/issues/44

**Comment by fschrempf** [comment]
Closing as duplicate of #44.
